ClosedXML/Excel/Cells/XLCellCopyOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
I think the introduction of `XLCellCopyOptions` should be its own commit, preferrably the first one, before sparklines are added.